backend/chunker/chunker.go

```diff
@@ -1352,6 +1352,20 @@ func (f *Fs) Purge(ctx context.Context, dir string) error {
 	return do(ctx, dir)
 }
 
+// Resume checks whether the (remote, ID) pair is valid and returns
+// the point the file should be resumed from or an error.
+//
+// This command will chain to `resume` from wrapped remote.
+// Can later be replaced by an implementaiton of `resume` for chunker
```

Comment:
@mcalman
we can't just chain to the lower backend in a general case. if the file is chunked it's remote chains to a small metadata (and to nothing if metadata is disabled). if it's non-chunked, it can become chunked after resume, but we can't predict it given arguments here.

i suggest that you either return unsupported error or remove interface from chunker.

@ncw 
you requested to add resume support to all wrapping backends. i'm not sure how this can be done for chunker.

Reply:
let's add the opt-out clause for chunker